product_price_category/models/product_pricelist_item.py

```diff
@@ -13,6 +13,10 @@ class ProductPricelistItem(models.Model):
         selection_add=[("2b_product_price_category", "Price Category")],
         ondelete={"2b_product_price_category": "set default"},
     )
+    display_applied_on = fields.Selection(
```

Comment:
@bealdav No need of migration script for this ?

Reply:
Thanks for your vigilance @rousseldenis 
I've added a migration script.I could not test it on a real migrated database though, but I did test that it runs without crashing